exonum-node/src/events/mod.rs

```diff
@@ -210,10 +220,10 @@ pub struct EventsAggregator<S1, S2, S3, S4> {
 
 impl<S1, S2, S3, S4> EventsAggregator<S1, S2, S3, S4>
```

Comment:
I suppose that we can replace this struct easier by utilising a new async/await syntax.

Reply:
How though? I think the reasons `EventsAggregator` is implemented like now is to provide differing priorities for constituent streams, which doesn't seem like an easy task to do using only "high-level" `async` / `await`.